base/regex.jl

```diff
@@ -334,6 +334,9 @@ See also [`occursin`](@ref) and [`endswith`](@ref).
 julia> startswith("JuliaLang", r"Julia|Romeo")
 true
 ```
+# See also
+[`endswith`](@ref), [`occursin`](@ref), [`match`](@ref)
+
```

Comment:
Adding `match` is a good idea, though it just needs to be added to the existing see also section above.

Same for the `endswith` docstring below